common/unsafe/src/test/java/org/apache/spark/unsafe/types/CollationSupportSuite.java

```diff
@@ -526,10 +526,40 @@ public void testInitCap() throws SparkException {
     assertInitCap("aB 世 de", "UNICODE_CI", "Ab 世 De");
     assertInitCap("ÄBĆΔE", "UNICODE_CI", "Äbćδe");
     // Case-variable character length
-    assertInitCap("İo", "UTF8_BINARY", "İo");
-    assertInitCap("İo", "UTF8_BINARY_LCASE", "İo");
-    assertInitCap("İo", "UNICODE", "İo");
-    assertInitCap("İo", "UNICODE_CI", "İo");
+    assertInitCap("İo", "UTF8_BINARY", "I\u0307o");
+    assertInitCap("İo", "UTF8_BINARY_LCASE", "İo");
+    assertInitCap("İo", "UNICODE", "I\u0307o");
```

Comment:
I don't understand why this has changed. It should not have.

Reply:
"İ" was 2 separate characters, it was only hard to see because of the way it's rendered in GitHub
these tests for UTF8_BINARY/UNICODE are only updated to properly reflect the diff wrt. UTF8_BINARY_LCASE

Reply:
> "İ" was 2 separate characters, 
We need to write it as two separate characters then, otherwise the rendering can be very misleading.

Reply:
yup, that's what changed here for UTF8_BINARY (and UNICODE - although this will get changed once we merge https://github.com/apache/spark/pull/46772), it's now write it in tests as 2 separate characters